src/rust/bitbox02-rust-c/src/util.rs

```diff
@@ -11,6 +11,17 @@ pub extern "C" fn rust_util_zero(mut dst: BytesMut) {
     util::zero(dst.as_mut())
 }
 
+#[no_mangle]
+pub extern "C" fn rust_util_all_ascii_bytes(bytes: Bytes) -> bool {
+    util::ascii::all_ascii(bytes)
+}
+
+#[no_mangle]
+pub extern "C" fn rust_util_all_ascii(cstr: CStr) -> bool {
+    let s: &str = cstr.as_ref();
+    util::ascii::all_ascii(s)
```

Comment:
I think this could also be written `util::ascii::all_ascii(&cstr)`.

Reply:
it can be, but i wanted to make the type clear if `s`. With all the traits/generics it can be hard to tell what the concrete type is at times.